src/hooks/hookProxy.ts

```diff
@@ -17,98 +18,80 @@ enum SupportedCodeSystems {
-                from:[EHRWhitelist.any]
-            }
-        ],
-        '1666386': [ // Addyi
-            {
-                to: REMSAdminWhitelist.standardRemsAdmin,
-                from:[EHRWhitelist.any]
-            },
-        ]
+const pbook = [
+    {
+        code: '6064',
+        system: 'http://www.nlm.nih.gov/research/umls/rxnorm',
+        to: REMSAdminWhitelist.standardRemsAdmin,
+        from: [EHRWhitelist.any]
     },
-    [SupportedCodeSystems.ndc]: {
-        '0245-0571-01': [ // iPLEDGE
-            {
-                to: REMSAdminWhitelist.standardRemsAdmin,
-                from:[EHRWhitelist.any]
-            }
-        ],
-        '63459-502-30': [ // TIRF
-            {
-                to: REMSAdminWhitelist.standardRemsAdmin,
-                from:[EHRWhitelist.any]
-            }
-        ],
-        '65597-402-20': [ // Turalio
-            {
-                to: REMSAdminWhitelist.standardRemsAdmin,
-                from:[EHRWhitelist.any]
-            }
-        ]
+    {
+        code: '1237051',
+        system: 'http://www.nlm.nih.gov/research/umls/rxnorm',
+        to: REMSAdminWhitelist.standardRemsAdmin,
+        from: [EHRWhitelist.any]
+    },
+    {
+        code: '2183126',
+        system: 'http://www.nlm.nih.gov/research/umls/rxnorm',
+        to: REMSAdminWhitelist.standardRemsAdmin,
+        from: [EHRWhitelist.any]
+    },
+    {
+        code: '1666386',
+        system: 'http://www.nlm.nih.gov/research/umls/rxnorm',
+        to: REMSAdminWhitelist.standardRemsAdmin,
+        from: [EHRWhitelist.any]
+    },
+    {
+        code: '0245-0571-01',
+        system: 'http://hl7.org/fhir/sid/ndc',
+        to: REMSAdminWhitelist.standardRemsAdmin,
+        from: [EHRWhitelist.any]
+    },
+    {
+        code: '63459-502-30',
+        system: 'http://hl7.org/fhir/sid/ndc',
+        to: REMSAdminWhitelist.standardRemsAdmin,
+        from: [EHRWhitelist.any]
+    },
+    {
+        code: '65597-402-20',
+        system: 'http://hl7.org/fhir/sid/ndc',
+        to: REMSAdminWhitelist.standardRemsAdmin,
+        from: [EHRWhitelist.any]
+    },
+]
+export async function loadPhonebook() {
+    const model = Connection;
+    for (const entry of pbook) {
+        const doesExist = await model.exists({ code: entry.code, to: entry.to });
+        if(!doesExist) {
+            const resource = new model(entry);
+            await resource.save();
+
+        } else {
+            console.log(`Skipping code ${entry.code} - already in database`);
+        }
     }
 }
-
-export const getServiceUrl = (coding: Coding, requester: string | undefined) => {
-    if(coding.system && coding.code && requester) {
-        const connections = phonebook[coding.system]?.[coding.code];
-        if(!connections){
+export async function getServiceUrl(coding: Coding, requester: string | undefined) {
+    const connectionModel = Connection;
+    if (coding.system && coding.code && requester) {
```

Comment:
When a hooks request comes from the SMART on FHIR app, the requester is undefined. We should be able to match the requester of undefined to the EHRWhitelist.any and send back the cards. I tested this and all you should need to do is remove the `&& requester` from the if statement above.

Reply:
I merged dev into this branch and made this change.